app/src/main/java/org/mifos/mobile/ui/activities/HomeActivity.kt

```diff
@@ -78,7 +80,7 @@ class HomeActivity :
     public override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         binding = ActivityHomeBinding.inflate(layoutInflater)
-        activityComponent?.inject(this)
+        //activityComponent?.inject(this)
```

Comment:
If this line of code is not required anymore then feel free to remove it.

Reply:
yes indeed , i must have missed it

Reply:
done